_tips/developing.ja.md

```diff
@@ -0,0 +1,642 @@
+  <div id="example-button-menu" class="menu" aria-hidden="true" tabindex="-1">
+    <ul>
+      <li><a href="javascript:return false">説明</a></li>
+      <li><a href="javascript:return false">ニュース</a></li>
+      <li><a href="javascript:return false">チケット</a></li>
+      <li><a href="javascript:return false">興味あるもの</a></li>
+    </ul>
+  </div>
+</div>
+<script>
+  !function() {
+    "use strict";
+    function e(e) {
+        "true" == e.getAttribute("aria-hidden") ? (e.setAttribute("aria-hidden", "false"), e.focus()) : e.setAttribute("aria-hidden", "true")
+    }
+    function t(e) {
+        "true" == e.getAttribute("aria-checked") ? (e.setAttribute("aria-checked", "false"), e.getElementsByTagName("span")[0].innerHTML = "Off") : (e.setAttribute("aria-checked", "true"), e.getElementsByTagName("span")[0].innerHTML = "On")
+    }
+    var a = document.getElementById("example-button");
+    a && (a.addEventListener("keydown", function(t) {
+        t.keyCode && 13 == t.keyCode && (e(document.getElementById("example-button-menu")), "true" == t.target.getAttribute("aria-expanded") ? t.target.setAttribute("aria-expanded", "false") : t.target.setAttribute("aria-expanded", "true"))
+    }), a.addEventListener("click", function(t) {
+        e(document.getElementById("example-button-menu")), "true" == t.target.getAttribute("aria-expanded") ? t.target.setAttribute("aria-expanded", "false") : t.target.setAttribute("aria-expanded", "true")
+    }));
+    var n = document.getElementById("switch");
+    n && (n.addEventListener("keydown", function(e) {
+        !e.keyCode || 32 != e.keyCode && 13 != e.keyCode || t(this)
+    }), n.addEventListener("click", function() {
+        t(this)
+    }))
+}();
+</script>
+</figure>
+<figure>
+<figcaption>{% include_cached icon.html name="code" %} Code Snippet</figcaption>
+{:/}
+
+~~~javascript
+var buttonExample = document.getElementById('example-button');
+
+buttonExample.addEventListener('keydown', function(e) {
+  // Toggle the menu when RETURN is pressed
+  if(e.keyCode && e.keyCode == 13) {
+    toggleMenu(document.getElementById('example-button-menu'));
+  }
+});
+
+buttonExample.addEventListener('click', function(e) {
+// Toggle the menu on mouse click
+  toggleMenu(document.getElementById('example-button-menu'));
+});
+~~~
+
+{::nomarkdown}
+</figure>
+</div>
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [キーボード 2.1.1](/WAI/WCAG21/quickref/#keyboard) ([理解のため 2.1.1](/WAI/WCAG21/Understanding/keyboard))
+* **ユーザーストーリー**
+  * [RSI(関節炎)患者がどのようにキーボードサポートを必要としているかを説明](/people-use-web/user-stories/#reporter)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## 可能な限りCAPTCHAを避ける
+
+CAPTCHAは多くの人々に問題を引き起こす｡ユーザー入力が人間によって生成されたものであることを検証するには､自動検出やインターフェイスのインタラクションなど､もっと使いやすい手段があります｡CAPTCHAを本当に含める必要がある場合は､理解しやすく､障害を持つユーザーのための代替手段を含むようにしてください｡下記にその例を示します｡
+
+* CAPTCHAを解く2つ以上の方法を提供すること｡
```

Comment:
@jaykishigami The original text is "Providing more than two ways to solve the CAPTCHAs" (i.e at least 3) and the translation seems to be "Providing two ways or more ways to solve the CAPTCHAs" or "more than one way".